### PR TITLE
Update Keycloak documentation

### PR DESCRIPTION
**Pull request by oplantalech**
* Update links
* Update commands, removing deprecated Docker images.
* Document changing the default credentials in Keycloak database container.

### CODE REVIEW

File: docs/using-keycloak.md [hunk 1]
```diff
@@ -32,12 +32,14 @@ docker run -d --restart=always \
 
 Then run the actual Keycloak server, using the MySQL-compatible
 version of
-[the Keycloak images](https://github.com/jboss-dockerfiles/keycloak)
+[the Keycloak images](https://hub.docker.com/r/jboss/keycloak/)
```
Comment (fedde-s) [inline docs/using-keycloak.md]: After this change, the link no longer refers to a list containing one MySQL-compatible version. Could you rephrase the sentence to reflect this, e.g. just calling it *the* image?

**Review by fedde-s — CHANGES_REQUESTED** [review]
Looks great and addresses the issues we discussed, although one sentence should now probably be simplified.

File: docs/using-keycloak.md (hunk 1 above)
Comment (oplantalech) [reply docs/using-keycloak.md]: Fixed :)

**Comment by fedde-s** [comment]
Not an unforgivable ‘error’ I'd say, but nice! Do you want me to squash the fixup commit?
